README.md

```diff
@@ -1,6 +1,5 @@
-# kubernetes-nmstate
-
 <img src="logo/fullcolor.png" alias="project logo" />
+# kubernetes-nmstate
```

Comment:
Since there is no space above this, this does not render as a title. Please move it on the top of the file as it was and maintain the empty line

Reply:
@qinqon ^^

Reply:
Ahh, sorry

Reply:
Done